app/services/record_declarations/base.rb

```diff
@@ -43,7 +43,9 @@ def call
       declaration.refresh_payability!
       declaration_attempt.update!(participant_declaration: declaration)
 
-      { id: declaration.id }
+      participant_declarations_hash = ParticipantDeclarationSerializer.new(declaration).serializable_hash
```

Comment:
Why introduce a new variable and then just to_json it?

Reply:
Done in 78054cc